packages/terra-scroll/src/terra-dev-site/doc/example/ScrollVertical.module.scss

```diff
@@ -0,0 +1,13 @@
+:local { 
+  .overflow-y-wrapper {
```

Comment:
We could combine `ScrollHorizontal.module.scss` and `ScrollVertical.module.scss` into a single scss file say `ScrollCommon.module.scss`. Since "overflow-x-wrapper" and "overflow-y-wrapper" have the same styles.